containers/Jars/useJarsWithAPYPoly.ts

```diff
@@ -224,24 +224,24 @@ export const useJarWithAPY = (network: ChainName, jars: Input): Output => {
 
       // Getting MATIC rewards
       const [
-        totalAllocPointCREncoded,
+        // totalAllocPointCREncoded,
         poolInfoCR,
         maticPerSecondBN,
       ] = await Promise.all([
-        provider.getStorageAt(MATIC_COMPLEX_REWARDER, 5),
+        // provider.getStorageAt(MATIC_COMPLEX_REWARDER, 5),
```

Comment:
This is for testing only. Interestingly, on Polygon this completely halts the evaluation of this hook. It's as if this particular call is evaluated as expensive and it won't return for a long time. I believe this is also happening a lot in production.